buildsystem/cpp.cmake

```diff
@@ -88,7 +89,7 @@ function(finalize_binary target_name output_name type)
 				message(FATAL_ERROR "finalize_binary flag 'allow_no_undefined' is only valid for libraries!")
 			endif()
 
-			if(NOT "${CMAKE_CXX_FLAGS}" MATCHES "-fsanitize")
+			if(NOT "${CMAKE_CXX_FLAGS}" MATCHES "-fsanitize" AND NOT MSVC)
```

Comment:
I'm confused. Undefined symbols would result in link-time errors.
Can you please elaborate what errors we're looking for?